kornia-py/src/image.rs

```diff
@@ -36,12 +35,10 @@ impl<const C: usize> FromPyImage<C> for Image<u8, C> {
             // TODO: we should find a way to avoid copying the data
             // Possible solutions:
             // - Use a custom ndarray wrapper that does not copy the data
-            // - Return direectly pyarray and use it in the Rust code
-            let data = unsafe {
-                match pyarray.as_slice() {
-                    Ok(d) => d.to_vec(),
-                    Err(_) => return Err(ImageError::ImageDataNotContiguous),
-                }
+            // - Return directly pyarray and use it in the Rust code
+            let data = match pyarray.to_vec() {
```

Comment:
@emilmgeorge tangential to #149 -- ideally we would like in python too to have zerop copies, especially that in the python case we need to allocate everytime the outputs, or find an anti-pattern for python to allow passing preallocated images :)